woocommerce/wc-admin
# Update non-wca pages registered with nav to embed pages

Updates pages using the new nav to become embed pages if they are not WCA pages.

Found while registering nav items in external plugins; WCA scripts and styles are not loaded and would need to be manually enqueued otherwise.

Embedding allows us not only to access scripts, but also adds some necessary page styling.  As @psealock noted, the header is also used during mobile/collapsed styling so this seems like the right path forward.

### Detailed test instructions:

1. Register nav items on a non-WCA and non-embed page. You can use this PR to test - https://github.com/woocommerce/woocommerce-memberships/pull/103
1. Make sure the nav loads.
1. Make sure no regressions occur on other admin pages.
